validator_derive/src/lib.rs

```diff
@@ -208,112 +211,153 @@ fn quote_field_validations(
 }
 
 /// Find if a struct has some schema validation and returns the info if so
-fn find_struct_validation(attr: &syn::Attribute) -> SchemaValidation {
+fn find_schema_validation(path: &syn::Path, nested: &syn::punctuated::Punctuated<syn::NestedMeta, syn::token::Comma>) -> SchemaValidation {
```

Comment:
This looks like a huge change but actually its just renaming the function and pulling the first 2 lines of the `if_chain` into the calling function.